numpy/random/tests/test_random.py

```diff
@@ -510,6 +512,25 @@ def test_shuffle_masked(self):
             assert_equal(
                 sorted(b.data[~b.mask]), sorted(b_orig.data[~b_orig.mask]))
 
+    @pytest.mark.parametrize("random",
+        [np.random, np.random.RandomState(), np.random.default_rng()])
```

Comment:
I am not sure that this is how we do things (I don't think it is right now), should we do this more, maybe aim from a more generic fixture when it comes up?

Reply:
This seems like a better way to test. For instance, there are tests called `test_beta` (two functions down in this file) reveals exact copies of the same test in `numpy/random/tests/test_generator_mt19937.py`. Using a fixture/parametrize could reduce the repeats.